### ISSUE TITLE
Allow easy use of command-line parameters

### ISSUE THREAD

**Issue by KalmanMeth** [netobserv/flowlogs-pipeline#110]
In the current implementation (with multiple possible stages of the same type), the command line parameters feature is no longer feasible. We need to design and implement a command-line parameter specification that can be used with the new multiple pipelines.

**Comment by KalmanMeth** [comment]
Also, if there are required parameters, force required fields in command line, such as an ingest stage.

**Comment by ronensc** [comment]
I suggest to follow Helm's approach for passing individual parameters from the command line.

In helm, the a yaml file is specified with `-f` flag. Multiple files and URLs are also supported.
Setting individual parameters is done with `--set`. Helm allows multiple parameters in a single flag by seperating them with a comma: `--set key1=val1,key2=val2`. It also allows multiple instances of the `--set` flag: `--set key1=val1 --set key2=val2`.

References:
- https://helm.sh/docs/chart_template_guide/values_files/
- https://github.com/technosophos/k8s-helm/blob/master/docs/using_helm.md#the-format-and-limitations-of---set

**Comment by eranra** [comment]
@ronensc I agree that this is an option although from my games with Helm this is not the easiest way in the world. .... but since I do not have any better idea I have to agree with that one. In any event this will not be part of this PR but will be part of future PR
